layouts/register/single.html

```diff
@@ -1,14 +1,10 @@
 {{ partial "head.html" . }}
 
```

Comment:
We could add a `{{ /* Hugo Comment */ }}` here that better describes what this is with documentation source.